documentation/asciidoc/services/connect/troubleshooting.adoc

```diff
@@ -10,6 +10,8 @@
 
 * Browsers intercept certain keys and key combinations. As a result, you can't type these keys into your Connect window. Screen sharing includes a toolbar to simulate some of the most popular intercepted keys.
 
+* Upgrading `rpi-connect` and `rpi-connect-lite` using Connect is not supported. The upgrade process will terminate all remote shell sessions and drop all connections.
```

Comment:
...but you can _usually_ reconnect to the same Pi using Connect once the upgrade process has completed?